worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -422,6 +422,13 @@ private[celeborn] class Worker(
         fileWriter.destroy(new IOException(
           s"Destroy FileWriter ${fileWriter} caused by shuffle ${shuffleKey} expired."))
       }
+      val expiredFileWrites =
+        partitionLocationInfo.getAllMasterLocations(shuffleKey).asScala.map(p =>
```

Comment:
We can delete the fileWriter when traverse the locations in the upper code block. mountPoint can be reconstructed from fileWriter.FileInfo.filePath, which is always ```${mountPoint}/${appId}/${shuffleId}/${fileName}```

Reply:
now filewriter is delete in locations traverse code